### DISCUSSION TITLE
为什么换成github了

### DISCUSSION THREAD

**Discussion by M453-ui** [Meloong-Git/PCL#1524] [提问]
为什么转到Github上进行交流？是不是有黑客入侵了我们群

**Comment by WForst-Breeze** [comment]
没看懂什么意思（）

这里是 PCL2 启动器的源码仓库，用于受理启动器相关反馈和建议，向开发者和社区协管寻求技术帮助或进行相关技术或非技术讨论。这里一般仅会交流关于启动器的内容，我没看懂您所说的……「黑客」是什么意思qwq

**Comment by Chiloven945** [comment]
~阴谋论是吧~
github比原来的金数据方便得多，而且pcl开源了还可以直接pr

**Comment by iPodtouch6557** [comment]
emmm…你多少年没看过反馈了，与世隔绝了吗（）

**Reply by Chiloven945** [reply]
别人就问一下情况你就开始在这写文章了
